azure_gateway/_chat.py

```diff
@@ -62,7 +63,13 @@ def openai_chat_request(project_id: str, params: OpenAIBody | dict, token: str):
 
 
 def parse_openai_chat_response(response: requests.Response) -> str:
```

Comment:
This function should be renamed since it parses the chat responses for `GenericGatewayLLM` as well. I think I had a chat with Lukas about this. As I've said in another comment, I've deprecated it instead of renaming it to not break backwards compatibility.